metadata-ingestion/src/datahub/ingestion/reporting/datahub_ingestion_run_summary_provider.py

```diff
@@ -200,18 +195,29 @@ def on_completion(
         report: Dict[str, Any],
         ctx: PipelineContext,
     ) -> None:
+        # Prepare a nicely formatted summary
+        structured_report_str = json.dumps(report, indent=2)
+        summary = f"~~~~ Ingestion Report ~~~~\n{structured_report_str}\n\n"
+        summary += "~~~~ Ingestion Logs ~~~~\n"
+        summary += get_log_buffer().format_lines()
+
         # Construct the dataHubExecutionRequestResult aspect
+        structured_report = StructuredExecutionReportClass(
+            type="CLI_INGEST",
+            serializedValue=structured_report_str,
+            contentType="application/json",
+        )
         execution_result_aspect = ExecutionRequestResultClass(
             status=status,
             startTimeMs=self.start_time_ms,
             durationMs=self.get_cur_time_in_ms() - self.start_time_ms,
-            report=json.dumps(report, indent=2),
+            report=summary,
```

Comment:
Nice! So in this case, we are now sending the structured report to the correct field. From here, we can continue to add to that report and get it into our result aspect

Reply:
(Sets ourselves up for much richer reports)

Reply:
yup exactly - this way the structured report shows up in the correct field for both managed ingestion and cli ingestion

note that the structured report field is basically a string right now - we may need to rethink that in the future